### ISSUE TITLE
what should i do?

### ISSUE THREAD

**Issue by 921712319** [cesarferreira/dryrun#59]
Searching for devices...
Killing adb, there might be an issue with it...
No devices attached, but I'll run anyway

======================================================

 Something went wrong while executing this:
  $ git clone --depth 1 https://github.com/shenhuanet/ViewScreenshot.git C:/User
s/ADMINI~1/AppData/Local/Temp/dryrun/5bb074c6fd3aa1e9dd140de65bbab485bd09f5faccc
677f9bc84a72ca610ab04

**Comment by 921712319** [comment]
and I used it in window.please let me know what you have any solutions.

**Comment by Yeluohui** [comment]
I have the same problem,need help

**Comment by luispereira** [comment]
When you type the command "adb devices" on the terminal what appears?

**Comment by cesarferreira** [comment]
Run this and tell me what happens:

```
git clone --depth 1 https://github.com/shenhuanet/ViewScreenshot.git C:/User
s/ADMINI~1/AppData/Local/Temp/dryrun/5bb074c6fd3aa1e9dd140de65bbab485bd09f5faccc
677f9bc84a72ca610ab04
```

**Comment by cesarferreira** [comment]
This was probably solved by #63
